docs/general/Mainnet/high_level_architecture.md

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 1
+---
+
+# High-level architecture
+
+![Architecture](./images/architecture.png)
+
+!!! . The node also gets informed from FFS about the finality status, hence the arrow should also go back.
+
+!!! . where is the DA light node in this diagram? i assume the DA box is actually two boxes - one for the DA service and one for the DA light node.?
+
+!!! . so Maptos  is the full node / validator node ?
+
+!!! . where is the follower node in this diagram?
+
+!!! . do we really send transactions 1-by-1 to the DA service? or do we send batches of transactions? if blobs, where are they created?
+
+## Data Availability
+
+The DA service is used to store transaction data and make it available to the network. The service is accessed via [DA light nodes](./node_level_architecture.md#da-light-node).
+
+Celestia is used as the Data Availability (DA) layer.
+
+
+!!! . do nodes pay fees to celestia? or is how is this rate limited?
```

Comment:
Should we create a MD for fees, which may well be empty at the moment and link here?

Reply:
Franck started a MIP: https://github.com/movementlabsxyz/MIP/pull/19/

I added a new section "fees" to address your comment